source/views/help/wifi-tools.js

```diff
@@ -14,13 +15,17 @@ export const getIpAddress = (): Promise<?string> =>
 	})
 
 export const getPosition = (args: any = {}): Promise<Object> =>
-	new Promise(resolve => {
-		navigator.geolocation.getCurrentPosition(resolve, () => resolve({}), {
-			...args,
-			enableHighAccuracy: true,
-			maximumAge: 1000 /*ms*/,
-			timeout: 5000 /*ms*/,
-		})
+	new Promise((resolve, reject) => {
+		if (Platform.OS === 'ios') {
+			navigator.geolocation.getCurrentPosition(resolve, reject, {
+				...args,
+				enableHighAccuracy: true,
+				maximumAge: 1000 /* ms */,
+				timeout: 15000 /* ms */,
+			})
+		} else {
```

Comment:
Nit: Would you flip these conditions, please? Android's needing to not have the third argument is very much the exception, I hope.